auth/src/main/java/com/vimeo/networking2/Authenticator.kt

```diff
@@ -31,31 +29,23 @@ class Authenticator(private val serverConfig: ServerConfig) {
      */
     private var retrofitServicesCache: RetrofitServicesCache
 
+    private var apiErrorResponseErrorConverter: ApiResponseErrorConverter
+
     init {
         retrofit = RetrofitSetupComponent.retrofit(serverConfig)
         retrofitServicesCache = RetrofitSetupComponent.retrofitCache(retrofit)
+        apiErrorResponseErrorConverter = RetrofitSetupComponent.apiResponseErrorConverter(retrofit)
```

Comment:
I don't think the `init` block is actually needed. I think you could simply init this in-line.